docs/xls-70d-credentials/index.md

```diff
@@ -0,0 +1,47 @@
+# Credentials
+
+The Credentials feature is a set of tools for managing authorization and compliance requirements using the XRP Ledger blockchain, while respecting privacy and decentralization. This feature extends and interconnects with other features of the XRP Ledger including [Decentralized Identifiers (DID)](https://xrpl.org/docs/concepts/accounts/decentralized-identifiers) and [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth). The goal of this feature is to streamline the process of compliance checks such as [KYC (Know Your Customer)](https://en.wikipedia.org/wiki/Know_your_customer) requirements and to enable further trust-based applications within the XRP Ledger ecosystem.
```

Comment:
Note: credentials don't directly connect with DIDs at all.

Reply:
Yeah, my thought was they interconnect _indirectly_ in that you can use a DID together with a Credential for authenticating to an off-chain service, but you're right that that's vague and theoretical enough to make this statement less than helpful.